rest/src/db/dbUtils.js

```diff
@@ -77,6 +79,18 @@ const dbUtils = {
 			return { [sortFieldDbRelation[options.sortField]]: { [1 === options.sortDirection ? '$gt' : '$lt']: offset } };
 		}
 		return undefined;
+	},
+
+	/**
+	 * Convert binary to Unresolved address
+	 * @param {MongoDb.Binary} binary Address|NamespaceId from MongoDb.
+	 * @returns {string} AddressBase32|NamespaceId
+	 */
+	bufferToUnresolvedAddress: binary => {
+		if (!(binary instanceof MongoDb.Binary))
```

Comment:
What happens if the binary is an alias? Like 0x99C1ED94FF2D65C0AF000000000000000000000000000000 ?

Reply:
Catbuffer allows aliases in the (unresolved)address fields and mongo stores it. Rest needs to render that, AAAA doesn't seem right to me...